set.mm

```diff
@@ -259089,34 +259089,40 @@ inner product (as a function) on the parent space.  (Contributed by NM,
 $)
 
   $c ocv $.
-  $c CSubSp $.
+  $c ClSubSp $.
   $c toHL $.
 
   $( Extend class notation with orthocomplement of a subspace. $)
   cocv $a class ocv $.
 
   $( Extend class notation with set of closed subspaces. $)
-  ccss $a class CSubSp $.
+  ccss $a class ClSubSp $.
 
   $( Extend class notation with the Hilbert lattice. $)
   cthl $a class toHL $.
 
   ${
     $d h s x y $.
-    $( Define orthocomplement of a subspace.  (Contributed by NM,
+    $( Define the orthocomplement function in a given pre-Hilbert space: it
```

Comment:
It is a little bit confusing that there is no pre-Hilbert space in the definition, also it is mentioned in the comment. Maybe we should write "Define the orthocomplement function for a set `h` which usually is a pre-Hilbert space".

We often have very general definitions, and make them "meaningful" in theorems by providing appropriate antecedents.

Reply:
I made the change (although I'm not sure it adds much information or is better). As you say the definition is too general. A better definition would be
```
df-ocv $a |- ocv = ( h e. Prehil |-> ...
```

Reply:
I just ran into https://us.metamath.org/mpeuni/ocvval.html and https://us.metamath.org/mpeuni/elocv.html, etc. whose comments say "pre-Hilbert space" whereas their statements are also not "guarded", that is, have antecedent/hypothesis of the form `e. V` instead of `e. PreHil`.

Reply:
I think for consistency, the comments for these should be changed, too. And also for df-css and df-thl.
In general, I think there was a general agreement to keep definitions as general as possible. Maybe @digama0 or @david-a-wheeler can explain this further.

Reply:
I think there is really no need for that (actually, I think I should go back to my first version for df-ocv. There is really nothing gained, nor any information added, by commenting (for https://us.metamath.org/mpeuni/df-ocv.html)
```
Define the orthocomplement function in a given set (which usually is a pre-Hilbert space)
```
instead of
```
Define the orthocomplement function in a given pre-Hilbert space
```
when one can see that the statement has the form `df-ocv $a |- ocv = ( h e. _V |-> ...`.

(The definition is a bit more general than for pre-Hilbert spaces of course: it could be pre-Hilbert module or any extensible structure that has defined the slots used in the definition.)

Reply:
To be honest, I think it's not that a big deal. Can we merge this now @avekens, and if you want, change the comments in a future PR and/or if we get an opinion from @digama0 ?